src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -103,27 +104,27 @@ export const useGetTradeQuotes = () => {
         swapperName: SwapperName.LIFI,
       },
     ].map(result => {
-      const quote = result.data && result.data.isOk() ? result.data.unwrap() : undefined
-      const inputOutputRatio = quote
-        ? getInputOutputRatioFromQuote({
-            quote,
-            swapperName: result.swapperName,
-          })
-        : -Infinity
-
+      const inputOutputRatio = selectInputOutputRatio(store.getState())
```

Comment:
this wont work because the selector is computing based on the currently selected quote, not the one in the result here

Reply:
Excellent catch. Derp.

https://github.com/shapeshift/web/pull/4782/commits/ff03b92e29699e757281d5948db6e783cb48ca52